forms-web-app/src/validators/appellant-submission/applicant-name.js

```diff
@@ -0,0 +1,19 @@
+const { body } = require('express-validator');
+
+const rules = () => {
+  return [
+    body('behalf-appellant-name')
+      .notEmpty()
+      .withMessage('Enter the name your appealing for')
+      .bail()
+      .matches(/^[a-z\-' ]+$/i)
+      .withMessage('Name must only include letters a to z, hyphens, spaces and apostrophes')
+      .bail()
+      .isLength({ min: 2, max: 255 })
+      .withMessage('Name must be between 2 and 255 characters'),
```

Comment:
I can't see these 3 `withMessage` entries on the error document or the ticket - not saying they are wrong, but if you haven't already done so, I'd get Marc or Daisy to confirm they are exactly what they want.

Reply:
As we said I didn't checked the document at the moment, so by default I reused the existing rule for the name, which might already be wrong so I guess we should fix these message once and for all with a specific PR.